documentation/modules/adding-source-provider.adoc

```diff
@@ -26,37 +26,166 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add Open Virtual Appliance (OVA) files that were created by or are compatible with VMware vSphere as source providers.
```

Comment:
you meant adding a provider (not files) or import files, right?

Reply:
@ahadas Adding OVA files created by VMware vSphere as a source provider. I can add the nuance about compatibility in a note, if that's better.